beamer/check/commands.py

```diff
@@ -478,3 +478,246 @@ def verify_l1_invalidations(
         sys.exit(1)
 
     log.info("Invalidations for all chain pairs successful", verified=len(verified))
+
+
+@dataclass
+class Challenge:
+    request_chain: ChainId = field(metadata=schema(min=1))
```

Comment:
as it is ChainId maybe we should name this like request_chain_id, same true for next line for fill_chain_iid

Reply:
Eh, I was doing following the same pattern as in the L1 invalidation-related commands... it's a bit shorter and perhaps a tiny bit more readable, but OTOH it does differ from the usual pattern in e.g. `beamer.agent`.
I'd prefer to leave it as is, if for nothing else, then for for the similarity with the existing check commands.